src/js/lang/en.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
   "Errors.ServiceUnavailable": "Unable to connect to the server. Please retry another time.",
 
   // Menu
-  "Menu.Bars.Tooltip": "Click to keep menu open"
+  "Menu.Bars.Tooltip": "Click to keep menu open",
+  "New Artefact": "New Artefact",
```

Comment:
Suggest keeping this as Pascal case with `.` for separators.

Reply:
Thank you, I will fix that.

Reply:
I am doing something wrong, when I change the entry in en.js to "New.Artefact": "New Artefact", the website now shows "New.Artefact" instead of the expected "New Artefact".